src/classes/player.js

```diff
@@ -0,0 +1,60 @@
+import {
+  GRAVITY, FRICTION, JUMPHEIGHT, MOVESPEED, CANVAS_HEIGHT,
+} from '../constants';
+
+class Player {
+  constructor() {
+    this.position = {
+      x: 100,
+      y: 100,
+    };
+    this.width = 100;
+    this.height = 100;
+    this.color = 'red';
```

Comment:
I would pass the starting position, width, height and color in the constructor.
It will be much easier to e.g. create other playable characters (or skins).